.buildkite/pipeline.yml

```diff
@@ -74,6 +112,8 @@ steps:
           run: payload-helper-tests
       - docker-compose#v3.7.0:
           run: docker-tests
+    env:
+      RUBY_VERSION: "3"
```

Comment:
Should this be `3.0`? If this is just `3` the version will change with every ruby release, but maybe that's what you want?

Reply:
That was my intention, on the assumption that users on Ruby are likely to follow the latest to a large extent - unless you think it's a bad idea for any reason.

Reply:
No, that makes sense 👍